dxr/templates/nunjucks/breadcrumbs.html

```diff
@@ -0,0 +1,11 @@
+{% macro breadcrumbs(www_root, paths_and_names) %}
+  <div class="breadcrumbs">
+    {%- for path, name in paths_and_names -%}
+      {%- if (not loop.first) or (loop.last and loop.length != 1) -%}
```

Comment:
> I guess I didn't really think about simplifying the expression, I just looked up the precedence rules and unpacked them...

Yeah, I did the same thing the first time around :-)

Thanks for all the updates!  Not sure if you're still planning on adding `@` processing to `Filter.__str__`?